src/src/DataStructs/SettingsStruct.h

```diff
@@ -198,6 +198,9 @@ class SettingsStruct_tmpl
   // Return hostname with explicit set append unit.
   String getHostname(bool appendUnit) const;
 
+  // Return the name of the unit, without unitnr appended, with template parsing applied, replacement for Settings.Name in most places
+  String getUnitname() const;
```

Comment:
OK, the description of this function does probably render half of my comments above to be useless.
But that's telling something about the badly named function name as it apparently doesn't do what the same suggests.

Hot sure what would be a better name here though.